rc/tools/go/gopls.kak

```diff
@@ -0,0 +1,49 @@
+hook -once global BufSetOption filetype=go %{
+    require-module gopls
+}
+
+evaluate-commands %sh{
+    if ! command -v gopls > /dev/null 2>&1; then
```

Comment:
I think this check should be done when the module is loaded.

Reply:
Thanks for your feedback! I have moved this part under the provide-module line.